packages/camera/camera_windows/windows/capture_controller.h

```diff
@@ -13,6 +13,7 @@
 #include <mfidl.h>
 #include <windows.h>
 #include <wrl/client.h>
+#include <flutter/event_channel.h>
```

Comment:
nit: Order includes alphabetically within groups (move this just above the `flutter/texture_register.h` include.